cli.js

```diff
@@ -4,29 +4,95 @@
 /* eslint no-console:0 */
 
 const katex = require("./");
-let input = "";
+const fs = require('fs');
 
 // Skip the first two args, which are just "node" and "cli.js"
 const args = process.argv.slice(2);
 
 if (args.indexOf("--help") !== -1) {
     console.log(process.argv[0] + " " + process.argv[1] +
                 " [ --help ]" +
-                " [ --display-mode ]");
+                " [ --display-mode ]" +
+                " [ --throw-on-error ]" +
+                " [ --error-color color ]" +
+                " [ --max-size size ]" +
+                " [ --input inputFile ]" +
+                " [ --output outputFile ]"
+               );
 
     console.log("\n" +
                 "Options:");
-    console.log("  --help            Display this help message");
-    console.log("  --display-mode    Render in display mode (not inline mode)");
+    console.log("  --help              Display this help message");
+    console.log("  --display-mode      Render in display mode (not inline mode)");
+    console.log("  --no-throw-on-error Render unsupported commands as text");
+    console.log("  --error-color       Color unsupported commands are rendered in");
+    console.log("  --max-size          User-specified sizes are capped to maxSize");
+    console.log("  --input file        Read from given file");
+    console.log("  --output file       Write to given file");
     process.exit();
 }
 
-process.stdin.on("data", function(chunk) {
-    input += chunk.toString();
-});
+function getOptions(args) {
```

Comment:
If you have time, would you be able switch the option parsing code to use `nomnom`.  We use this in a couple of other places in the project, e.g. https://github.com/Khan/KaTeX/blob/master/dockers/Screenshotter/screenshotter.js.

Reply:
I did not want to introduce a new dependency. Using a proper option parsing library will make things much easier.

Reply:
Done

Reply:
Hmm: nomnom is deprecated: https://github.com/harthur/nomnom

Reply:
Thanks for switching to nomnom.  That's too bad about it's deprecation.  I created #900 to switch from nomnom to commander.js everywhere.  It's not super pressing b/c nomnom does the job.